src/coreclr/src/vm/siginfo.cpp

```diff
@@ -988,7 +988,8 @@ TypeHandle SigPointer::GetTypeHandleThrowing(
                  BOOL                        dropGenericArgumentLevel/*=FALSE*/,
                  const Substitution *        pSubst/*=NULL*/,
                  // ZapSigContext is only set when decoding zapsigs
-                 const ZapSig::Context *     pZapSigContext) const
+                 const ZapSig::Context *     pZapSigContext,
```

Comment:
Does this really need a new argument everywhere?

I would expect that you can get what you need by initializating `ZapSig::Context` with your multicore-JIT specific callbacks.

Reply:
Thank you. I will fix it.